numba/tests/is_comparison_tests.py

```diff
@@ -0,0 +1,63 @@
+import unittest
```

Comment:
You don't have to create a new file for this test. You can put your tests inside the class `TestArrayComparisons` on `numba/tests/test_array_methods.py`. There is already a test case for array comparison there. Just do the same for record.